.env.sample

```diff
@@ -8,4 +8,7 @@ OSM_CONSUMER_SECRET=your-oauth-secre
 OSM_DOMAIN=https://www.openstreetmap.org
 SESSION_SECRET=super-secret-sessions
 DATABASE_URL=postgres://scoreboard:test@localhost:5433/scoreboard
-OSM_TEAMS_SERVICE=http://localhost:3000
\ No newline at end of file
+OSM_TEAMS_SERVICE=http://localhost:8989
+OSM_TEAMS_SERVICE_TOKEN_URL=http://localhost:4445
+OSM_TEAMS_CONSUMER_KEY=teams-oauth-key
```

Comment:
It would be convenient for `OSM_TEAMS_CONSUMER_KEY` instead be named `OSM_TEAMS_CLIENT_ID`.
Same for `OSM_TEAMS_SECRET`, it would instead be `OSM_TEAMS_CLIENT_SECRET`.

Keeping the same naming scheme as the issuing app would help decrease second-guessing about whether the implementer is setting the right variables.